handmade-example-strings.adoc

```diff
@@ -0,0 +1,681 @@
+.[[D3Example]]<<D3Example,D3>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('ui-d3')">Focus Inspector</button>
+<div class="short narrow example" id="ui-d3"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/ui/d3_example.cljs[]
+----
+
+====
+
+
+;;;;;;;;;;;;;;;;;;;
+
+.[[SampleExample]]<<SampleExample,Sample Example>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('example-1')">Focus Inspector</button>
+<div class="short narrow example" id="example-1"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/example_1.cljs[]
+----
+
+====
+
+
+;;;;;;;;;;;;;;;;;;;
+
+
+
+.[[]]<<,Input Focus and React Refs/Lifecycle>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('focus-example')">Focus Inspector</button>
+<div class="short narrow example" id="focus-example"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/ui/focus_example.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Drawing in a Canvas>>
+====
```

Comment:
A lot of these look corrupt to me

Reply:
Yup, these might already be resolved in the latest push submitted here

Reply:
Great, yeah, that looks better :)